db/controller.js

```diff
@@ -63,13 +63,24 @@ exports.patchArticle = (req, res, next) => {
 
     if (!inc_votes) {
         return res.status(400).send({ msg: 'Bad request' });
-      }
+    }
 
     updateArticleVotes(article_id, inc_votes)
-    .then((updatedVotes) => {
-        if(!updatedVotes){
-            return res.status(400).send({ msg: "Bad request" });
-        }
-        res.status(200).send({ article : updatedVotes });
-    })
+        .then((updatedVotes) => {
+            if (!updatedVotes) {
+                return res.status(400).send({ msg: "Bad request" });
+            }
+            res.status(200).send({ article: updatedVotes });
+        })
```

Comment:
All of your controllers should have a catch block to catch any errors that have occurred.